### PR TITLE
feature: Config-ifies a bunch of Hunger-related values. Balance tweaks.

### PR DESCRIPTION
**Pull request by scrdest**
1. Most Hungers give more Suppression by default now. 
2. Made Vamp fangs actually usable for blood-drinking (aim for the head). 
3. Made regen much more powerful (up to a *full-on revive*) but more expensive and does NOT work on fire wounds (because both vamps and The Thing are weak to fire so it fits).

### CODE REVIEW

File: code/modules/urist/antagonist/revenant/archetypes/vampiric.dm [hunk 1]
```diff
@@ -372,7 +372,7 @@
 
 /* FANGS - gives mobs a new unarmed attack that requires head and ignores restraints. */
 /datum/unarmed_attack/bsrevenant_vampbite
-	attack_verb = list("bit", "sank his fangs in")
+	attack_verb = list("bit", "sank their fangs in")
```
Comment (Glloyd) [inline code/modules/urist/antagonist/revenant/archetypes/vampiric.dm:375]: you can use the /his macro here

File: baystation12.dme [hunk 2]
```diff
@@ -3634,10 +3634,10 @@
 #include "interface\skin.dmf"
 #include "maps\_map_include.dm"
 #include "maps\_maps.dm"
-#include "maps\away\icarus\icarus_areas.dm"
 #include "maps\away\geminus_city\road_and_pavements.dm"
 #include "maps\away\geminus_city\signs.dm"
 #include "maps\away\geminus_city\streetobjects.dm"
+#include "maps\away\icarus\icarus_areas.dm"
```
Comment (Glloyd) [inline baystation12.dme:3640]: this shouldn't be part of this pr

separately, icarus_areas needs to be moved to icarus.dm but @CPU-Blanc may have already done that in an upcoming pr

Comment (CPU-Blanc) [reply baystation12.dme:3640]: `icarus_areas` is already included in `icarus.dm`, without any PRs or anything like that.

Comment (Glloyd) [reply baystation12.dme:3640]: definitely should be removed from the dme entirely in that case, if you want to just do that here @scrdest

Comment (scrdest) [reply baystation12.dme:3640]: Oh, sure. IDK what's up with that, it's the DME acting weird again and it's not even that the IDE was open this time

File: code/modules/urist/antagonist/revenant/archetypes/vampiric.dm (hunk 1 above)
Comment (scrdest) [reply code/modules/urist/antagonist/revenant/archetypes/vampiric.dm:375]: Ah, cool, wasn't sure when it resolves and how

**Comment by Glloyd** [comment]
if you resolve the conflicts and the things mentioned above today, i can have it in for this weekend.

File: code/modules/urist/antagonist/revenant/archetypes/chimera.dm [hunk 3]
```diff
@@ -101,51 +101,103 @@
...
+				if(valid_revenant)
+					revenant.total_distortion += BSR_DISTORTION_GROWTH_OVER_SECONDS(clone_dmg, BSR_DEFAULT_DISTORTION_PER_TICK, BSR_DEFAULT_DECISECONDS_PER_TICK)
+					return FALSE
+
+		// Tox
+		if(pending)
+			var/tox_dmg = L.getToxLoss()
+			if(tox_dmg)
+				L.adjustToxLoss(-tox_dmg)
+				to_chat(L, SPAN_NOTICE("You purge absorbed toxins from your cells (though not your bloodstream)!"))
```
Comment (Naxodile) [inline code/modules/urist/antagonist/revenant/archetypes/chimera.dm:180]: For humans toxloss is just damage to organs (excluding the brain) nowadays.

File: code/modules/urist/antagonist/revenant/archetypes/chimera.dm [hunk 4]
```diff
@@ -101,51 +101,103 @@
...
+		// First things first - if they are dead, revive
 		if(pending)
-			for(var/obj/item/organ/I in H.internal_organs)
-				if(I.damage >= 5)
-					I.damage = max(I.damage - 5, 0)
-					to_chat(H, SPAN_NOTICE("Your [I.name] itches as you knit its wounds shut."))
-					pending = FALSE
-					break
+			if(H.stat == DEAD)
+				H.adjustOxyLoss(-rand(15,20))
```
Comment (Naxodile) [inline code/modules/urist/antagonist/revenant/archetypes/chimera.dm:118]: You probably want to bump this up since anyone dead's going to have 200 oxyloss and will get pretty significant brain damage by the time they're awake.

File: code/modules/urist/antagonist/revenant/archetypes/chimera.dm [hunk 5]
```diff
@@ -101,51 +101,103 @@
...
+				continue
 
-			for(var/datum/wound/W in E.wounds)
-				if(W.bleeding())
-					W.bleed_timer = 0
-					E.status &= ~ORGAN_BLEEDING
-					to_chat(H, SPAN_NOTICE("You knit together severed veins, stemming the bleeding from your [E.name]."))
-					pending = FALSE
-					break
+			E.rejuvenate()
```
Comment (Naxodile) [inline code/modules/urist/antagonist/revenant/archetypes/chimera.dm:137]: This is going to also full heal all organs inside each limb. bad_external_organs also includes any limb with internal damage (excluding brain damage) even if it lacks external damage.

**Comment by Glloyd** [comment]
Nax the goddess of medical saving the day yet again. thank you.

File: code/modules/urist/antagonist/revenant/archetypes/chimera.dm (hunk 3 above)
Comment (scrdest) [reply code/modules/urist/antagonist/revenant/archetypes/chimera.dm:180]: Yeah, but I'm accounting for **non**humanoids, potentially.

File: code/modules/urist/antagonist/revenant/archetypes/chimera.dm (hunk 4 above)
Comment (scrdest) [reply code/modules/urist/antagonist/revenant/archetypes/chimera.dm:118]: This value is actually stolen from Bay's psionics revive code. Keep in mind, this power is meant to potentially be used multiple times in a row - obviously, for a higher total cost in realityfuckery - so brain damage can be healed by that too.

File: code/modules/urist/antagonist/revenant/archetypes/chimera.dm (hunk 3 above)
Comment (Naxodile) [reply code/modules/urist/antagonist/revenant/archetypes/chimera.dm:180]: Besides maybe the phoron pistol I'm not sure there's many instances of simple mobs or any other living mobs taking toxin damage. It's just going to be a kind of strange message for most circumstances.

File: code/modules/urist/antagonist/revenant/archetypes/chimera.dm (hunk 5 above)
Comment (scrdest) [reply code/modules/urist/antagonist/revenant/archetypes/chimera.dm:137]: That's kind of the point; I would do that by individual organs as prior except, err, that code didn't work properly fsr and would routinely block all other healing (which is why there's a prob onnit; stochastically it will eventually heal everything, just potentially on multiple tries)

File: code/modules/urist/antagonist/revenant/archetypes/vampiric.dm (hunk 1 above)
Comment (scrdest) [reply code/modules/urist/antagonist/revenant/archetypes/vampiric.dm]: Done

File: baystation12.dme (hunk 2 above)
Comment (scrdest) [reply baystation12.dme]: Should be sorted

File: code/modules/urist/antagonist/revenant/archetypes/chimera.dm (hunk 3 above)
Comment (scrdest) [reply code/modules/urist/antagonist/revenant/archetypes/chimera.dm]: Adjusted

File: code/modules/urist/antagonist/revenant/archetypes/chimera.dm (hunk 5 above)
Comment (Naxodile) [reply code/modules/urist/antagonist/revenant/archetypes/chimera.dm:137]: Since neither this or the blood restoration trigger any distortion increases burn damage is healed for free if you just cast this enough.

Comment (Naxodile) [reply code/modules/urist/antagonist/revenant/archetypes/chimera.dm:137]: Never mind I missed the 300 seconds catch all increase. The cost being detached from the amount of damage healed should still be considered though.

Comment (scrdest) [reply code/modules/urist/antagonist/revenant/archetypes/chimera.dm:137]: Nope, they do - that's what the fallthrough case at the very bottom does. Unless a heal-type has a specific custom cost, it gives it a generic cost.

**Comment by Glloyd** [comment]
I'll move BSR to votable for next weekend so we can get more chances to test the new values.
